mobilecoind/strategies/drain-accounts.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2018-2021 The MobileCoin Foundation
+
+"""
+The purpose of this strategy is to drain funds from one account to another.
+This is one way to transfer tokens to a fog account, which were created after bootstrap.
+This is the case for minted tokens (which fog distro cannot transfer).
+
+It zips the source accounts and dest accounts together, and each source account sends
+its entire balance (less fee) to the dest account
+
+Example setup and usage:
+```
+    python3 drain-accounts.py --key-dir ../../../target/sample_data/master/keys/ --dest-key-dir ../../../target/sample_data/master/fog_keys/ --fee 20
+```
+"""
+import argparse
+import concurrent.futures
+import grpc
+import mobilecoind_api_pb2
+import mobilecoind_api_pb2_grpc
+import os
+import time
+from accounts import *
```

Comment:
Better to use explicit import 

```suggestion
from accounts import wait_for_accounts_sync
```

Reply:
thank you!

i got this by copying `test_client.py`, i will try to mirror those changes onto that one